pandas/tests/scalar/conftest.py

```diff
@@ -0,0 +1,6 @@
+import pytest
+
+
+@pytest.fixture(name="td_overflow_msg")
+def fixture_td_overflow_msg() -> str:
+    return R"outside allowed range \[-9223372036854775807ns, 9223372036854775807ns\]"
```

Comment:
lower case r, also pretty sure this can just be a constant instead of a fixture

Reply:
Can switch 👍 (Since this repo includes a `devcontainer.json`, I thought maybe use `R` for [VS Code (and GitHub) compatibility](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#r-strings-and-r-strings).)